Dockerfile

```diff
@@ -23,13 +23,13 @@ RUN cmake --build _build -j $(nproc)
 RUN cmake --install _build
 RUN rm -rfv _build
 
-FROM ubuntu:22.04 AS runner
+FROM ubuntu:24.04 AS runner
 
 RUN date
 
 RUN apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
-        libsqlite3-0 libtiff5 libcurl4 libcurl3-gnutls \
+        libsqlite3-0 libtiff6 libcurl4 libcurl3-gnutls \
```

Comment:
Surprisingly in line 16 the dev package is `libtiff5-dev`. There is no `libtiff6-dev` in Ubuntu 24.04

Reply:
Looking at package descriptions, [`libtiff5-dev`](https://packages.ubuntu.com/noble/libtiff5-dev) is a transitional package that effectively points to [`libtiff-dev`](https://packages.ubuntu.com/noble/libtiff-dev). It's a low priority to update transitional packages to their primary target.